.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
 script: make travis
```

Comment:
I understand your view on that. I hesitate because in 9000 other things I do, it's "make ..." so it's basically muscle memory for me. For my own sanity I try to keep things consistent.